block/block.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://schemas.wp.org/trunk/block.json",
+  "apiVersion": 3,
+  "name": "team51/colophon",
+  "version": "0.1.0",
+  "title": "Colophon",
+  "category": "common",
+  "icon": "editor-textcolor",
+  "description": "A block for displaying a colophon.",
+  "supports": {
+    "html": false,
+    "align": true,
+    "typography": {
+      "fontSize": true,
+      "lineHeight": true,
+      "__experimentalFontFamily": true,
+      "__experimentalFontStyle": true,
+      "__experimentalFontWeight": true,
+      "__experimentalLetterSpacing": true,
+      "__experimentalTextTransform": true,
+      "__experimentalDefaultControls": {
+        "fontSize": true
+      }
+    }
+  },
+  "textdomain": "team51-blocks",
+  "editorScript": "file:./index.js",
+  "editorStyle": "file:./index.css",
+  "style": "file:./style-index.css",
+  "render": "file:./render.php"
+}
```

Comment:
I know block context can be used to set a block where it can only be used in a post context or the like --

https://developer.wordpress.org/block-editor/reference-guides/block-api/block-context/

Is there an option we can set so that the block is only usable in a FSE / page template context, so we're not cluttering up the block options when someone's writing a post?

Reply:
Blocks meant for use in FSE themes are called Theme Blocks, and I think this would be the way to do it:

https://github.com/WordPress/gutenberg/blob/e706d67c7482e752a4adaae3e0b0449234e19902/packages/block-library/src/navigation/block.json#L6

Reply:
Currently, even if using `theme` as the category for the theme block. It is still visible in the post editor. The only option I can think of now is to [remove the block programatically](https://developer.wordpress.org/news/2024/01/29/how-to-disable-specific-blocks-in-wordpress/) on the post editor. Should we go that route?